app/views/custom_actions/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<% html_title l(:label_administration), t('custom_actions.plural') -%>
+<%= toolbar title: t('custom_actions.plural') do %>
+  <li class="toolbar-item">
+    <%= link_to new_custom_action_path,
+                { class: 'button -alt-highlight',
+                  aria: { label: t('custom_actions.new') },
+                  title: t('custom_actions.new') } do %>
+      <%= op_icon('button--icon icon-add') %>
+      <span class="button--text"><%= CustomAction.model_name.human %></span>
+    <% end %>
+  </li>
+<% end %>
+<div class="generic-table--container">
```

Comment:
We can avoid repeating the table styles everywhere when we re-use the `TableCell` and `RowCell`. Simply subclass both. This would make the next table refactoring much simpler. You can find a basic example here: https://github.com/opf/openproject-ldap_groups/tree/dev/app/cells/ldap_groups/synchronized_groups